src/components/ThreadSummary.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+	<div class="summary">
+		<div class="summary__header">
+			<div class="summary__header__title">
+				<span class="summary__header__title__brand">
+					<CreationIcon class="summary__header__title__brand__icon" />
+					<p>Nextcloud Assistant</p>
```

Comment:
@jancborchardt @nimishavijay what do you think about this? should it come from theming, and/or should it be translated ?

Reply:
I would say yes, without theming it would look confusing to users who have never interacted with the Nextcloud brand on their instance. And yes it should be translated. But double checking with @jancborchardt

Reply:
Yep, what @nimishavijay said! :)